fpdf/table.py

```diff
@@ -1,13 +1,13 @@
 from dataclasses import dataclass
 from numbers import Number
+from types import NoneType
```

Comment:
This only exists in Python 3.10, and `fpdf2` supports Python 3.7 to 3.11

You could try something like this:
```python
try:
    from types import NoneType
except ImportError:
    NoneType = type(None)
```